okrs_2019.md

```diff
@@ -0,0 +1,209 @@
+# SourceCred 2019 OKRs
+
+This document contains SourceCred's objectives, initiatives, and metrics for
+2019, generated as part of Protocol Lab's annual planning process.
+For an overview of SourceCred itself, see [the overview](overview.md).
+
+The purpose of this document is to provide clarity around what the project's
+goals and priorities are, and accountability to see if we are making effective
+progress towards achieving those goals. However, this is not intended to lock
+us in to a particular set of initiatives--we know that as time goes on, new
+opportunities will present themselves, and some of the existing initiatives
+will prove ill-founded or infeasible. Also, our metrics have known deficiencies
+(e.g. we don't yet know how to measure the quality of a cred attribution).
+
+As such, this is intended to be a living document. We will update it both to
+record progress on our goals, and to modify and retarget our goals as we gain
+more information. Since this document is in Git, all revisions will be
+publically visible. If we remove or replace a goal, initiave, or metric, we'll
+clearly explain the rationale for doing so in the associated commit.
+
+
+# SourceCred Goals
+
+## Improve Attribution Quality
+
+SourceCred's core function is to produce cred attributions for open-source
```

Comment:
> Improving the quality of those attributions is the our highest
> priority

It is good to note this explicitly. I hope that it continues to remain
the top priority.